src/panelApi.js

```diff
@@ -6,22 +6,76 @@ import Debug from 'debug'
 
 const debug = Debug('pdk:panelapi')
 
+function parseJwt (token) {
+  return JSON.parse(Buffer.from(token.split('.')[1], 'base64').toString());
+}
+
+function getTokenRefreshInterval (token) {
+  const tokenPayload = parseJwt(token)
```

Comment:
The `jose` library is already imported by the `openid-client` lib, it throws `JWTInvalid` if the token is not a JWT.

```suggestion
  const tokenPayload = jose.decodeJwt(token)
```

Reply:
@joshperry Looking up the release notes of [openid-client](https://github.com/panva/node-openid-client), I observed that `jose` started to be used by the lib version 3.0.0 (`node-pdk` uses `openid-client` v2.5.0) whereas `node-jose` was used before. From what I can see, `node-jose` does not have utils for working with JWT. I will install `jose` separately then.

Reply:
I agree this is the correct solution with either version of openid-client, I think it makes sense to add `jose` to `dependencies` if we are going to directly depend on it. I mentioned this because if its already a transitive dependency then it would not be adding any additional code in the dependency graph.

It probably also makes sense to update openid-client anyway as the current version is the target of two high and two moderate severity security vulnerabilities, since it is targeting the openid standard updates shouldn't regress compatibility.